src/main/java/com/loofah/graph/api/models/database/Category.java

```diff
@@ -57,6 +62,38 @@ public String getDescription() {
     }
 
 
+    @Override
+    public String toString() {
+        return "Category{" +
+                "id='" + id + '\'' +
+                ", title='" + title + '\'' +
+                ", description='" + description + '\'' +
+                '}';
+    }
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        final Category category = (Category) o;
+        return Objects.equals(id, category.id) &&
+                Objects.equals(title, category.title) &&
+                Objects.equals(description, category.description);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, title, description);
+    }
+
+    @Override
+    public int compareTo(@NotNull final Category otherCategory) {
```

Comment:
whatre your thoughts on having a little comment on this method (and the same in the other classes) to briefly explain what it does? "naturalOrder" for example isnt clear to me straight away what order that is (assuming alphabetical?)
I've not done loads with ordering though so if you think this is overkill / adding clutter then that is fine with me too

Reply:
Great idea, have added comments to describe each comparison